lib/aws-record/record/attributes.rb

```diff
@@ -26,6 +26,19 @@ def initialize
         @data = {}
       end
 
+      # @private
```

Comment:
I'm happy with the refactor, but why not move this below the existing `private` demarcation?